qiskit/transpiler/passes/layout/noise_adaptive_layout.py

```diff
@@ -55,11 +55,12 @@ class NoiseAdaptiveLayout(AnalysisPass):
          by being set in `property_set`.
     """
 
-    def __init__(self, backend_prop):
+    def __init__(self, backend_prop, coupling_map=None):
         """NoiseAdaptiveLayout initializer.
 
         Args:
             backend_prop (Union[BackendProperties, Target]): backend properties object
+            coupling_map (CouplingMap): Optional. To filter the backend_prop qubits/gates.
```

Comment:
I would add a note here that this argument is ignored if `backend_prop` is a `Target` object and this is only needed to deal with inconsistencies in `BackendProperties` objects.

Reply:
Extending it in 29bc0075b17605aedc832ae177635a9ece61074d